### PR TITLE
Add href prop for FontAwesomeIconButton and minor style fixes for Avatar and Section

### PR DESCRIPTION
**Pull request by andrewhillcode**
**The following MUST be checked prior to approval. If not relevant to your PR, remove the line from your description.**
- [ ]  The `styleguide.config.js` has been updated to show examples/new functionality for the component
- [ ]  Test Coverage for any new or updated functionality
- [ ]  New and updated components have been tested locally in lapis and SPA and work as expected
- [ ]  This PR has been tagged with PATCH, MINOR, or MAJOR.
- [ ]  The component has data-component-type and data-component-context attributes following the standard
- [ ]  You have requested a cross-team review on this component so everyone knows it exists

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -71,7 +71,7 @@ pipeline {
           script {
             try {
               sh '''
-                git fetch --no-tags --progress git@github.com:intellihr/ui-components.git \
+                git fetch --no-tags --progress git@github.com:intelliHR/ui-components.git \
```
Comment (KaitlynPickard) [inline Jenkinsfile:74]: why did this change?

**Review by KaitlynPickard — COMMENTED** [review]
LGTM

File: Jenkinsfile (hunk 1 above)
Comment (KaitlynPickard) [reply Jenkinsfile:74]: I don't care as long as it doesn't break anything

Comment (andrewhillcode) [reply Jenkinsfile:74]: the github pages deploy was broken, this should fix it.

File: src/domain/Avatars/Avatar/Avatar.scss [hunk 2]
```diff
@@ -33,7 +33,6 @@ $status-dot-styles: (
 }
 
 :local(.Avatar) {
-  display: inline-block;
```
Comment (wyvl) [inline src/domain/Avatars/Avatar/Avatar.scss:36]: will it affect the existing avatar?

Comment (andrewhillcode) [reply src/domain/Avatars/Avatar/Avatar.scss:36]: I didn't see any change with any of the avatar components

**Pull request merged**